lunatrace/bsl/backend/src/github/install.ts

```diff
@@ -113,10 +122,10 @@ export async function githubInstall(req: Request, res: Response): Promise<void>
     return;
   }
 
-  const installError = await waitForGithubInstall(installationId);
+  const installError = await waitForGithubInstall(installationId, installationAuthToken.res);
 
   if (installError.error) {
-    log.error('unable to verify that Github App was installed successfully', {
+    log.error(installError.msg, {
```

Comment:
we should keep the specific log message that was removed and put the key `error: installError.msg` in the object of this logging call. This will help us track down where an error is coming from since we don't have line numbers when we log.